mlflow/azure/client.py

```diff
@@ -0,0 +1,81 @@
+from copy import deepcopy
```

Comment:
Can we add a small docstring to this module explaining that it provides utilities for performing Azure Blob Storage operations without requiring the heavyweight `azure-storage-blob` library dependency?